beacon-chain/sync/rpc_blob_sidecars_by_root.go

```diff
@@ -37,13 +37,14 @@ func (s *Service) blobSidecarByRootRPCHandler(ctx context.Context, msg interface
 
 	blobIdents := *ref
 	cs := s.cfg.clock.CurrentSlot()
+	remotePeer := stream.Conn().RemotePeer()
 	if err := validateBlobByRootRequest(blobIdents, cs); err != nil {
-		s.cfg.p2p.Peers().Scorers().BadResponsesScorer().Increment(stream.Conn().RemotePeer())
+		s.cfg.p2p.Peers().Scorers().BadResponsesScorer().Increment(remotePeer)
 		s.writeErrorResponseToStream(responseCodeInvalidRequest, err.Error(), stream)
 		return err
 	}
 	// Sort the identifiers so that requests for the same blob root will be adjacent, minimizing db lookups.
-	sort.Sort(blobIdents)
+	sort.Sort(&blobIdents)
```

Comment:
Why did these change? `blobIdents` is a slice:
`type BlobSidecarsByRootReq []*eth.BlobIdentifier`
You should not have to pass a pointer?

Reply:
Fixed in 1f6680ef622ab5d15a2d1e89eb39332183846946.